runtime/call_metrics.go

```diff
@@ -132,6 +141,49 @@ var knownStatusCodes = map[int]bool{
 	http.StatusNetworkAuthenticationRequired: true, // 511
 }
 
+type systemErrorMap map[tchannel.SystemErrCode]tally.Counter
+
+func newsystemErrorMap(scope tally.Scope, key string) systemErrorMap {
+	ret := make(systemErrorMap)
+	ret[tchannel.ErrCodeInvalid] = scope.Counter(key)
+	for errorCode := range knownTchannelErrors {
+		ret[errorCode] = scope.Tagged(map[string]string{
+			"error": errorCode.MetricsKey(),
+		}).Counter(key)
+	}
+	return ret
+}
+
+// IncrErr will increase according error counter
+func (sem systemErrorMap) IncrErr(err error, count int64) {
+	errCause := tchannel.GetSystemErrorCode(errors.Cause(err))
+	counter, ok := sem[errCause]
+	if !ok {
+		counter = sem[0x00]
```

Comment:
instead of `0x00`, here the key should be `tchannel.ErrCodeInvalid`.

Reply:
`ErrCodeInvalid` commented as `"never should be used"`, show my respect for the author's will ;)